From dbbbb73bc017c4d9956b5349b9cd273f22457339 Mon Sep 17 00:00:00 2001
From: Dalon Westergreen <dwesterg@altera.com>
Date: Thu, 19 Nov 2015 15:56:36 -0600
Subject: [PATCH 171/172] FogBugz #336543: fix compatible string for SPI1

commit "f1ed0f4c FogBugz #250531-2: DTS: Update SPI node to align with
kernel.org" accidently added "ssnps", and it should be "snps" for the
SPI1 node.

Signed-off-by: Dalon Westergreen <dwesterg@altera.com>
Signed-off-by: Dinh Nguyen <dinguyen@opensource.altera.com>
[czou:Original patch taken from
https://github.com/altera-opensource/linux-socfpga.git socfpga-4.1]
Signed-off-by: czou <cao.zou@windriver.com>
---
 arch/arm/boot/dts/socfpga.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/socfpga.dtsi b/arch/arm/boot/dts/socfpga.dtsi
index 420120e..b77b553 100644
--- a/arch/arm/boot/dts/socfpga.dtsi
+++ b/arch/arm/boot/dts/socfpga.dtsi
@@ -825,7 +825,7 @@
 		};
 
 		spi1: spi@fff01000 {
-			compatible = "ssnps,dw-apb-ssi";
+			compatible = "snps,dw-apb-ssi";
 			#address-cells = <1>;
 			#size-cells = <0>;
 			reg = <0xfff01000 0x1000>;
-- 
1.9.1

